wizard/src/components/mapping_column_options.tsx

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Component for showing and updating options for a single column.
+ */
+
+import _ from "lodash";
+import React from "react";
+import { FormGroup, Input, Label } from "reactstrap";
+
+import { DCProperty, MappedThing, MappingType } from "../types";
+import { PlaceDetector } from "../utils/detect_place";
+import { ColumnInfo } from "./mapping_section";
+
+interface MappingColumnOptionsProps {
+  column: ColumnInfo;
+  onColumnUpdated: (column: ColumnInfo) => void;
+  validPlaceTypeProperties: Record<string, Set<DCProperty>>;
+  placeDetector: PlaceDetector;
+}
+
+const MAPPED_THING_NAMES = {
+  [MappedThing.UNIT]: "unit of measure",
+  [MappedThing.VALUE]: "observation value",
+};
+
+export function MappingColumnOptions(
+  props: MappingColumnOptionsProps
+): JSX.Element {
+  const validPlaceTypes = Object.keys(props.validPlaceTypeProperties);
+  const validPlaceProperties = Array.from(
+    props.validPlaceTypeProperties[props.column.columnPlaceType.dcid]
+  );
+  return (
+    <div className="column-options">
+      <div className="column-options-header">
+        The values in the column <b>{props.column.column.header}</b> are:
+      </div>
+      <FormGroup radio="true" className="column-options-input-section">
+        <Label radio="true" className="column-options-input column-type">
```

Comment:
Good to add a comment above each <Label> or <Input> to indicate what it is for, since all of them look very similar